ArrtModel/ViewModel/ApplicationModel.cpp

```diff
@@ -26,7 +26,7 @@
 #pragma clang diagnostic ignored "-Wunused-parameter"
 #pragma clang diagnostic ignored "-Wsign-compare"
 #pragma warning(push)
-#pragma warning(disable : 4100)
+#pragma warning(disable : 4100 4189)
```

Comment:
which one is 4189?

Reply:
unreferenced local variable, unfortunately the RemoteRendering.inl is full of that. It's auto-generated code, so it cannot easily be removed, but there is already an ApiGen ticket for that